aQute.libg/src/aQute/lib/home/Home.java

```diff
@@ -0,0 +1,21 @@
+package aQute.lib.home;
+
+public class Home {
+	public static final String	USER_HOME_BND_DEFAULT	= "~/.bnd";
+	public static final String	USER_HOME_BND_ENV		= "bnd.home.dir";
+
+	public static final String BND_USER_HOME() {
+		return USER_HOME(USER_HOME_BND_ENV, USER_HOME_BND_DEFAULT);
+	}
+
+	public static final String	USER_HOME_MVN_DEFAULT	= "~/.m2";
+	public static final String	USER_HOME_MVN_ENV		= "maven.home.dir";
```

Comment:
I cannot find any reference to this in the maven documentation. And I don't think Bnd should be creating system property names in the `maven` namespace. That namespace belongs to maven.

Reply:
Also, since this would only be a Bnd thing, we cannot influence maven itself. 

You only seem to use it for maven settings and not or the maven local repository. So the current PR is only partial.

In any case, I think if maven defines a standard system property (like `maven.repo.local` which we already use to find the `~/.m2/repository` folder), then we must use the maven standard way and not define a bnd way.

Perhaps you should limit this PR to just defining a way to specific the location of the bnd folder and leave maven things for another PR?

Reply:
`git grep '~/\.bnd'` finds a number of places which reference the default folder which this PR does not seem to address.

Reply:
`git grep '~/\.bnd' ` does only reference documentations, cli output, and readme files